libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -12,15 +12,17 @@ import (
 	"github.com/opencontainers/runc/libcontainer/cgroups"
 	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
 	"github.com/opencontainers/runc/libcontainer/configs"
-	"github.com/opencontainers/runc/libcontainer/system"
 )
 
 const (
 	cgroupCpuacctStat   = "cpuacct.stat"
 	nanosecondsInSecond = 1000000000
-)
 
-var clockTicks = uint64(system.GetClockTicks())
+	// The value comes from `C.sysconf(C._SC_CLK_TCK)`, and
+	// on Linux it's a constant which is safe to be hard coded,
+	// so we can avoid using cgo here.
```

Comment:
Please add link to `https://github.com/containerd/cgroups/pull/12#issuecomment-311327696`

Reply:
🤦 I had it there, but thought it was redundant; let me update

Reply:
Added